.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 test.py
 *.py.bak
+.env*
```

Comment:
Was this change _supposed_ to be included in this PR?

Reply:
yeah, it shouldn't be included here.